apps/rush-lib/src/cli/actions/VersionAction.ts

```diff
@@ -48,9 +50,14 @@ export default class VersionAction extends BaseRushAction {
     });
     this._ensureVersionPolicy = this.defineFlagParameter({
       parameterLongName: '--ensure-version-policy',
-      parameterShortName: '-e',
```

Comment:
>parameterShortName: '-e', [](start = 6, length = 25)

Do we need any of these short names?  I already updated all our CI definitions to always use long names.

Reply:
That line is deleted.

---
In reply to: [183606000](https://github.com/Microsoft/web-build-tools/pull/619#discussion_r183606000) [](ancestors = 183606000)